store-implementation/hbase-store/src/main/java/uk/gov/gchq/gaffer/hbasestore/serialisation/ElementSerialisation.java

```diff
@@ -194,8 +200,6 @@ public Properties getProperties(final String group, final Cell cell) throws Seri
                     } catch (final SerialisationException e) {
                         throw new SerialisationException(e.getMessage(), e);
                     }
-                } else {
-                    return serialiser.serialiseNull();
```

Comment:
What happens now ?

Reply:
The previous return is no longer hit so instead it goes down to `return HBaseStoreConstants.EMPTY_BYTES;`